baremetal/metal3machine_manager.go

```diff
@@ -1707,3 +1728,22 @@ func (m *MachineManager) getNodesWithLabel(ctx context.Context, nodeLabel string
 	}
 	return nodes, nodesCount, err
 }
+
+func getValueFromCM(m *MachineManager, ctx context.Context, configmapKey client.ObjectKey) string {
+	original := &corev1.ConfigMap{}
+
+	if err := m.client.Get(ctx, configmapKey, original); err != nil {
+		m.Log.Info("Failed to extract ConfigMap")
```

Comment:
could this be "failed to get configmap" ? to make it different from the error on line 1741

Reply:
done